src/channel/ably/utils.ts

```diff
@@ -33,18 +33,37 @@ export const toTokenDetails = (jwtToken: string): TokenDetails | any => {
 
 const isBrowser = typeof window === 'object';
 
+/**
+ * Helper method to decode base64 url encoded string
+ * https://stackoverflow.com/a/78178053
+ * @param base64 base64 url encoded string
+ * @returns decoded text string
+ */
 export const toText = (base64: string) => {
+    const base64Encoded = base64.replace(/-/g, '+').replace(/_/g, '/');
+    const padding = base64.length % 4 === 0 ? '' : '='.repeat(4 - (base64.length % 4));
```

Comment:
Not sure if we really need to add padding as per 
https://gist.github.com/tatsuyasusukida/ce71456081748242a0bd4cbfcfe44eb7

Reply:
I believe padding it's not necessary, message will be decoded without padding, but I think it's better to keep it, since padding is a part of base64  spec

Reply:
thanks, this is helpful